app/views/validators/index.html.erb

```diff
@@ -3,7 +3,12 @@
   <meta name="turbolinks-visit-control" content="reload">
 <% end %>
 
-<%= render 'validators/banner' %>
+<%= render 'validators/welcome_banner' %>
```

Comment:
why are these changes here ?

Reply:
I'll be needed in the next step. This cards should be separated for alignment.